slither.config.json

```diff
@@ -10,6 +10,5 @@
   ],
   "exclude_dependencies": true,
   "compile_force_framework": "hardhat",
-  "ignore_compile": true,
-  "skip_clean": true
```

Comment:
slither doesn't recognize this key

Reply:
are you sure? I explicitly remember adding this. It's passed through to cryptic cimpile if im not mistaken and removes a redundant step.

https://github.com/crytic/crytic-compile/blob/master/crytic_compile/crytic_compile.py#L552

Reply:
https://github.com/crytic/slither/discussions/1866#discussioncomment-5740034